scripts/pre_commit_linter.py

```diff
@@ -108,6 +109,14 @@
             'core/templates/dev/head/expressions/typeParserSpec.js')}
 }
 
+BAD_PATTERNS_JS_REGEXP = [
+    {
+        'regexp': r"\b(ddescribe|iit|fit|fdescribe)\(",
+        'message': 'Please do not use focused tests in this file.',
```

Comment:
Maybe break into two sets with different regexes:
- In tests, please use 'describe' instead of 'ddescribe' or 'fdescribe'
- In tests, please use 'it' instead of 'iit' or 'fit'

That makes it clearer for devs to figure out what to do when they get the error, I think.